lib/ui/painting/codec.cc

```diff
@@ -135,11 +235,15 @@ void InstantiateImageCodec(Dart_NativeArguments args) {
     return;
   }
 
-  Dart_Handle callback_handle = Dart_GetNativeArgument(args, 1);
-  if (!Dart_IsClosure(callback_handle)) {
-    TRACE_FLOW_END("flutter", kInitCodecTraceTag, trace_id);
-    Dart_SetReturnValue(args, ToDart("Callback must be a function"));
-    return;
+  if (image_info) {
+    int expected_size = image_info->minRowBytes() * image_info->height();
+    if (list.num_elements() < expected_size) {
+      TRACE_FLOW_END("flutter", kInitCodecTraceTag, trace_id);
+      list.Release();
```

Comment:
Isn't this implicit in the desctructor? I think you can get rid of this.

Reply:
The list needs to be explicitly released because Dart will not allow calls to APIs like Dart_SetReturnValue while holding a typed data buffer

Reply:
TIL